engine/src/test/java/com/redhat/cloud/notifications/recipients/rbac/RbacRecipientUsersProviderTest.java

```diff
@@ -41,7 +41,7 @@ public class RbacRecipientUsersProviderTest {
     @ConfigProperty(name = "recipient-provider.it.max-results-per-page", defaultValue = "1000")
     int maxResultsPerPage;
 
-    @ConfigProperty(name = "recipient-provider.rbac.elements-per-page", defaultValue = "1000")
+    @ConfigProperty(name = "user-provider.rbac.elements-per-page", defaultValue = "1000")
```

Comment:
This change doesn't look needed as nothing related to that property changed in `engine`.

Reply:
True, reverted!